source/manual/alerts/runs-rake-task-search-api.html.md

```diff
@@ -0,0 +1,25 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Runs a rake task on Search API that generates the sitemap files
+parent: "/manual.html"
+layout: manual_layout
+section: Icinga alerts
+last_reviewed_on: 2020-01-14
+review_in: 6 months
+---
+
+At 2:30am every day, a [Jenkins job][jenkins] runs a rake task on search-api
```

Comment:
The links to the job aren't working for me - looks like a markdown formatting issue.

Reply:
Ah yeah it's the parentheses around the URL in the footnote link.

Reply:
Whoops! Think it's the parentheses on line 25. Good spot!